lms/static/js/spec/staff_debug_actions_spec.js

```diff
@@ -97,7 +97,7 @@ define([
                     };
                     StaffDebug.doInstructorDashAction(action);
                     AjaxHelpers.respondWithTextError(requests);
-                    expect($('#idash_msg').text()).toBe('Failed to reset attempts for user. ');
+                    expect($('#idash_msg').text()).toBe('Failed to reset attempts for user. Unknown Error Occurred.');
```

Comment:
Is there anything more useful than Unknown Error we can give?

Reply:
This was because the newer version of jquery that I had karma point to changed its default behavior around a null error text. Rather than change any actual code, I just added the new text that jquery gives.